CIS560-RecipeManager/CIS560-RecipeManager/Recipe/Recipe.cs

```diff
@@ -30,5 +31,25 @@ public Recipe(int id, string name)
             Name = name;
             MeasuredIngredients = new Dictionary<Ingredient, int>();
         }
+
+        bool CookRecipe()
+        {
```

Comment:
I wonder if this method would be more suited for the RecipeController class?  Cooking the Recipe will involve updating the database as well as the Pantry class.  I was thinking that design-wise, we could have each Controller class hold a reference to the IQuery and call the database update methods as needed.  

Additionally, maybe we shouldn't worry about returning a boolean.  The purpose of having the user cook a recipe is to remove Ingredients from their pantry and if for some reason the Pantry is off (probably because the user didn't update it correctly when they shopped), I would think we'd want to just set the minimum value of an Ingredient as zero and not worry about returning anything.